scss/slider/_layout.scss

```diff
@@ -259,16 +262,29 @@ $slider-track-thickness: 4px !default;
         text-indent: -3333px;
         box-sizing: content-box;
 
+        $slider-draghandle-offset: -(($slider-draghandle-size - ($slider-track-thickness / 2)) / 2) !default;
+        $slider-draghandle-active-offset: -(($slider-draghandle-active-size - ($slider-track-thickness / 2)) / 2) !default;
+
         .k-slider-horizontal & {
-            height: 14px;
-            top: -6px;
-            width: 14px;
+            height: $slider-draghandle-size;
+            top: $slider-draghandle-offset;
+            width: $slider-draghandle-size;
+
+            &:active,
+            &.k-pressed {
+                top: $slider-draghandle-active-offset;
+            }
         }
 
         .k-slider-vertical & {
-            height: 14px;
-            left: -6px;
-            width: 14px;
+            height: $slider-draghandle-size;
+            left: $slider-draghandle-offset;
+            width: $slider-draghandle-size;
```

Comment:
seems we have the same width / height declarations for both vertical and horizontal handles, these can be moved to the draghandle in general.